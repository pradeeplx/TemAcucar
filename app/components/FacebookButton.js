import React from 'react'
import Colors from "../Colors"
import Button from "./Button"

export default FacebookButton = (props) => (
  <Button
    activityIndicatorColor={Colors.white}
    {...props}
    style={[{
      backgroundColor: Colors.facebook,
      borderColor: Colors.facebook,
    }, props.style]}
    textStyle={[{
      color: Colors.white,
    }, props.textStyle]}
  >
    {props.children}
  </Button>
)
