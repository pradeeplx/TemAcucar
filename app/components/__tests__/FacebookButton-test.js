import 'react-native'
import React from 'react'
import FacebookButton from '../FacebookButton'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <FacebookButton
      href="#"
      textStyle={{color: 'red'}}
    >
      FooBar
    </FacebookButton>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
