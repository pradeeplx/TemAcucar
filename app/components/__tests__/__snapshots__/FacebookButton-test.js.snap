exports[`test renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "#557BD3",
      "borderColor": "#557BD3",
      "borderRadius": 27,
      "borderWidth": 0.5,
      "flexDirection": "row",
      "height": 54,
      "justifyContent": "center",
      "marginBottom": 0,
      "opacity": 1,
      "paddingHorizontal": 30,
      "paddingVertical": 0,
    }
  }
  testID={undefined}>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
          "fontSize": 18,
          "textAlign": "center",
        },
        Array [
          Object {
            "color": "#FF7F85",
            "fontFamily": "Avenir",
            "fontSize": 18,
            "textAlign": "center",
          },
          Array [
            Object {
              "color": "#FFFFFF",
            },
            Object {
              "color": "red",
            },
          ],
        ],
      ]
    }>
    FooBar
  </Text>
</View>
`;
